pkg/gadgets/trace/capabilities/types/types.go

```diff
@@ -34,16 +34,20 @@ const (
 type Event struct {
 	eventtypes.Event
 
-	Pid       uint32 `json:"pid,omitempty" column:"pid,template:pid"`
-	Comm      string `json:"comm,omitempty" column:"comm,template:comm"`
-	Syscall   string `json:"syscall,omitempty" column:"syscall,template:syscall"`
-	UID       uint32 `json:"uid,omitempty" column:"uid,minWidth:6"`
-	Cap       int    `json:"cap,omitempty" column:"cap,width:3,fixed"`
-	CapName   string `json:"capName,omitempty" column:"capName,width:18,fixed"`
-	Audit     int    `json:"audit,omitempty" column:"audit,minWidth:5"`
-	Verdict   string `json:"verdict,omitempty" column:"verdict,width:7,fixed"`
-	InsetID   *bool  `json:"insetid,omitempty" column:"insetid,width:7,fixed,hide"`
-	MountNsID uint64 `json:"mountnsid,omitempty" column:"mntns,template:ns"`
+	Pid           uint32 `json:"pid,omitempty" column:"pid,template:pid"`
+	Comm          string `json:"comm,omitempty" column:"comm,template:comm"`
+	Syscall       string `json:"syscall,omitempty" column:"syscall,template:syscall"`
+	UID           uint32 `json:"uid,omitempty" column:"uid,minWidth:6"`
+	Cap           int    `json:"cap,omitempty" column:"cap,width:3,fixed"`
+	CapName       string `json:"capName,omitempty" column:"capName,width:18,fixed"`
+	Audit         int    `json:"audit,omitempty" column:"audit,minWidth:5"`
+	Verdict       string `json:"verdict,omitempty" column:"verdict,width:7,fixed"`
+	InsetID       *bool  `json:"insetid,omitempty" column:"insetid,width:7,fixed,hide"`
+	MountNsID     uint64 `json:"mountnsid,omitempty" column:"mntns,template:ns"`
+	TargetUserNs  uint64 `json:"targetuserns,omitempty" column:"targetuserns,template:ns"`
+	CurrentUserNs uint64 `json:"currentuserns,omitempty" column:"currentuserns,template:ns"`
+	Caps          uint64 `json:"caps,omitempty" column:"caps,hide"`
+	CapsNames     string `json:"capsNames,omitempty" column:"capsnames,hide"`
```

Comment:
Could it make sense to use `[]string` and let each "frontend" to format as they want?